### ISSUE TITLE
TEanno.sum generation

### ISSUE THREAD

**Issue by wyim-pgl** [oushujun/EDTA#169]
Hi Shujun,
Is there any way to get TEanno.sum from TEanno.gff3
Thanks

**Comment by oushujun** [comment]
Yes, you can. The easiest is to rerun EDTA with `--anno 1 --step anno`.

If you want to challenge yourself by making a sum file from a random gff3 file, this is a good start:

`perl ../util/gff2bed.pl  $genome.EDTA.TEanno.gff3 > $genome.EDTA.TEanno.bed`;

`perl -nle 'my (\$chr, \$s, \$e, \$anno, \$dir, \$supfam)=(split)[0,1,2,3,8,12]; print "10000 0.001 0.001 0.001 \$chr \$s \$e NA \$dir \$anno \$supfam"' $genome.EDTA.TEanno.bed > $genome.EDTA.TEanno.out`;

`perl ../util/buildSummary.pl -maxDiv 40 -genome_size $genome_size -seq_count $seq_count $genome.EDTA.TEanno.out > $genome.EDTA.TEanno.sum 2>/dev/null`;

Basically, you need to make an enriched bed file from the gff3 file, then convert the bed file to a fake RepeatMasker .out file, then use the summary script modified from RepeatMasker to produce the .sum file.

Best,
Shujun
